package.json

```diff
@@ -43,5 +43,13 @@
     "eslint-plugin-promise": "4.2.1",
     "eslint-plugin-standard": "4.0.1",
     "lerna": "3.20.2"
+  },
+  "dependencies": {
+    "apollo-link-error": "^1.1.12",
+    "apollo-link-retry": "^2.2.15",
+    "apollo-link-ws": "^1.0.19",
+    "graphql": "^14.6.0",
+    "ngx-pipes": "^2.7.3",
+    "subscriptions-transport-ws": "^0.9.16"
```

Comment:
Install the project specific dependencies  in your SPA folder only

Reply:
you mean i should install them in inner package.json ?

Reply:
Yes package.json of the feedback SPA